package/package.json

```diff
@@ -20,6 +20,5 @@
     ],
     "website": "https://docs.mesosphere.com/current/usage/service-guides/spark/",
     "selected": true,
-    "version": "$version",
-    "minDcosReleaseVersion": "1.7"
+    "version": "{{package-version}}"
```

Comment:
Script is here: https://github.com/mesosphere/dcos-commons/blob/master/tools/release_builder.py
Can be invoked in jenkins here (result: diff posted to Universe): https://jenkins.mesosphere.com/service/jenkins/job/infinity-release-to-universe/build?delay=0sec

Just borrowed someone's cluster to see what happened with a stub universe whose value was set to 1.7, didn't throw any errors so it should be good to put back (though keep in mind it'll be ignored when installed in stub-universe form). Done.